ee/desktop/runner/runner.go

```diff
@@ -516,7 +533,7 @@ func (r *DesktopUsersProcessesRunner) writeDefaultMenuTemplateFile() {
 func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
 	if !r.processSpawningEnabled {
 		// Desktop is disabled, kill any existing desktop user processes
-		r.killDesktopProcesses()
+		r.killDesktopProcesses(context.Background())
```

Comment:
Is this intentionally starting a new contenxt? (or should it be `context.TODO()`)